Shopify/Views/HomeScreen/HomeVC.swift

```diff
@@ -8,12 +8,105 @@
 import UIKit
 
 class HomeVC: UIViewController {
+    
+    @IBOutlet weak var OfferCV: UICollectionView!
+    @IBOutlet weak var BrandsCV: UICollectionView!
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
-    }
+        
+        OfferCV.delegate = self
+        OfferCV.dataSource = self
+        
+        BrandsCV.dataSource = self
+        BrandsCV.delegate = self
 
+        
+        let nib = UINib(nibName: "BrandCVCell", bundle: nil)
+        OfferCV.register(nib,forCellWithReuseIdentifier: "offerbrandcell")
+        BrandsCV.register(nib, forCellWithReuseIdentifier: "offerbrandcell")
```

Comment:
Don't use the same cell for both !! 
Add new files in "Cells" folder for the coupons...